universal-application-tool-0.0.1/conf/messages

```diff
@@ -85,6 +85,18 @@ button.submit=Submit
 # Continue application button
 button.continue=Continue
 
+# The text that appears next to the first unanswered question that the applicant can click on to continue filling out the program.
+link.begin=Start here
+
+# The text that appears next to an answered question that the applicant can click on to modify the response.
+link.edit=Edit
+
+# Program Prewiew Title
+title.programPreview=Program preview
```

Comment:
suggestion: Thoughts on using "Program application preview" and "Program application review"? Since these pages are showing the _application_ information rather than info about the program itself.

Reply:
TBH, I never understood why they're referred to as _Programs_ and prefer the term _Application_, so I'm happy to make the change.

Reply:
Ohh cool! I noticed that you changed the name from "Application summary" to "Program review", so I wasn't sure which one we were going with.